src/core/DataFile.cpp

```diff
@@ -467,7 +467,17 @@ bool DataFile::copyResources(const QString& resourcesDir)
 				{
 					// Get absolute path to resource
 					bool error;
-					QString resPath = PathUtil::toAbsolute(el.attribute(*res), &error);
+					auto attribute = el.attribute(*res);
+
+					// Skip empty resources. In some cases, there might be an
+					// alternative way of actually representing the data (e.g. data element for SampleTCO)
+					if (attribute.isEmpty())
```

Comment:
Is there a more specialized solution to this which may not cause regressions in completely different areas?